index.html

```diff
@@ -16,32 +17,171 @@
       <a href="#"> <img src="images/hamb.png" alt="hamb" class="humb"> </a>
     </nav>
   </header>
-  <section class="intro">
-    <div class="head">
-      <h1>
-        I’m Lumato
-        Glad to see you!
-      </h1>
-    </div>
-
-    <section class="content">
-      <p>I’m a software developer! I can help you build a
-        product , feature or website Look through some
-        ce! If you like what you see and have a project you need code
-        d, don’t hestiate to contact me.</p>
-      <p class="connect">LETS CONNECT</p>
+  <section class="whole">
+    <section class="intro">
+      <div class="head">
+        <h1>
+          I’m Lumato
+          Glad to see you!
+        </h1>
+      </div>
+
+      <section class="content">
+        <p>I’m a software developer! I can help you build a
+          product , feature or website Look through some
+          ce! If you like what you see and have a project you need code
+          d, don’t hestiate to contact me.</p>
+        <p class="connect">LETS CONNECT</p>
+      </section>
+      <section class="icons">
+        <ul>
+          <li><a href="#"><img src="images/twitter.png" alt="twitter"></a></li>
+          <li><a href="#"><img src="images/in.png" alt="in"></a></li>
+          <li><a href="#"><img src="images/M.jpg" alt="M"></a></li>
+          <li><a href="#"><img src="images/cat.png" alt="cat"></a></li>
+          <li><a href="#"><img src="images/fingers.png" alt="fingers"></a></li>
+        </ul>
+      </section>
```

Comment:
Kindly decrease the space between each section 

#### Original design
![right2](https://user-images.githubusercontent.com/58376020/121733983-76dda400-caf4-11eb-8b5f-d9845ebc40ba.png)

#### Your project
![wrong1](https://user-images.githubusercontent.com/58376020/121734024-8230cf80-caf4-11eb-87f1-6a9ec49796ea.png)

Reply:
- Not addressed

Reply:
how? i just changed the spacing and now there is enough spacing